src/index.bundlespec.js

```diff
@@ -9,11 +9,15 @@ describe('exports', () => {
     expect(customProperties).toBeTruthy();
   });
 
-  it('should have the translations', () => {
-    // This test ensures some translations exist, assuming the others would
-    // work as well then
-    expect(i18n.en['UIKit.DateInput.labelRange']).toEqual('to');
-    expect(i18n.de['UIKit.DateInput.labelRange']).toEqual('bis');
-    expect(i18n.es['UIKit.DateInput.labelRange']).toEqual('hasta');
+  it('should have translations', () => {
+    expect(Object.keys(i18n.en).length > 0).toBe(true);
```

Comment:
hm, does this need to be part of this PR?

Reply:
The reason this is here is because until now the tests were using `UIKit.DateInput.labelRange` to check whether the translations in the final bundle work. But in this PR we are dropping the `UIKit.DateInput.labelRange` message completely, so this test broke. To me it makes sense to change it here. Could be its own PR as well though, but 🤷‍♂️